esy/bin/esyCommand.ml

```diff
@@ -1190,8 +1190,11 @@ let getSandboxSolution installSandbox =
   return solution
 
 let solve {CommonOptions. installSandbox; _} () =
+  let open EsyInstall in
   let open RunAsync.Syntax in
   let%bind _ : Solution.t = getSandboxSolution installSandbox in
+  let unused = Resolver.getUnusedResolutions installSandbox.resolver in
```

Comment:
Let's move this into `getSandboxSolution` — it's called from `addCommand` too and we want to see warnings there as well.